### DISCUSSION TITLE
Contribute a landscape?

### DISCUSSION THREAD

**Discussion by AenghusOg** [Stellarium/stellarium#3839] [General]
This page
[https://stellarium.org/en/landscapes.html]
suggests that users may contribute a landscape. 
But I do not find link where the landscape package could be uploaded.
Is there such a link?
Thank you.

**Comment by gzotti** [comment]
See instructions on that page: Either send one of us the full package, or just the description with a link to your own website.  Be sure about your license.

**Comment by AenghusOg** [comment]
How do I “send one of you the package?”

Don’t I need an email address?

Thanks!

Howard


> On Aug 3, 2024, at 12:33 PM, Georg Zotti ***@***.***> wrote:
> 
> 
> See instructions on that page: Either send one of us the full package, or just the description with a link to your own website. Be sure about your license.
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>

Howard Goldbaum
***@***.***

https://howardgoldbaum.com
